demo/Makefile

```diff
@@ -18,5 +18,7 @@ stack-run:
 	godot -e --path ./game
 stack-watch:
 	stack build --file-watch --fast --exec "cp $(STACKLIBFILE) $(GODOTPROJECT)/lib"
+project-watch:
+	stack exec godot-haskell-parse-game game src
```

Comment:
stack exec godot-haskell-project-generator game src

Reply:
This should be fixed in the new version. Thanks!